tactile_calibration/tactile_calibration_tools/src/tactile_calibration_tools/calibration_utils.py

```diff
@@ -5,7 +5,6 @@
 import matplotlib.pyplot as plt
 from datetime import datetime
 import os
-#pip install pwlf
```

Comment:
Could we not keep that ? it was not left there by mistake. It helps co-workers...